dmsite/frontend/src/home/ClassificationColumn/ClassificationColumn.jsx

```diff
@@ -13,7 +13,7 @@ class ClassificationColumn extends Component {
 
   // TODO: fix this
   isSensitive(val) {
-    if (val == true) { // == works, but === doesn't, for bools or numbers - need to fix
+    if (val === true) { // == works, but === doesn't, for bools or numbers - need to fix
```

Comment:
The only thing that worries me about this is that when I was testing, the === wasn't working, so I'm not sure if this will still work. We can do more testing later though.

Reply:
oh the warning was causing the travis to fail so I changed it. My bad i total missed the comment

Reply:
ill change it back